enterprise/metrics/pom.xml

```diff
@@ -66,6 +66,8 @@
                   <arg value="org.neo4j.metrics.source.cluster.ClusterMetrics" />
 
                   <arg value="org.neo4j.metrics.source.jvm.JvmMetrics" />
+
+                  <arg value="org.neo4j.metrics.source.CoreEdgeMetrics" />
```

Comment:
I think `JvmMetrics` should be last since it is a bit special. Since we are here, can you add a comment about that?